core/src/main/java/tech/pegasys/web3signer/core/util/Eth1AddressSignerIdentifier.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2023 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.core.util;
+
+import static org.web3j.crypto.Keys.getAddress;
+import static tech.pegasys.signers.secp256k1.EthPublicKeyUtils.toHexString;
+import static tech.pegasys.signers.secp256k1.api.util.AddressUtil.remove0xPrefix;
+
+import tech.pegasys.signers.secp256k1.api.SignerIdentifier;
+
+import java.security.interfaces.ECPublicKey;
+import java.util.Locale;
+import java.util.Objects;
+
+public class Eth1AddressSignerIdentifier implements SignerIdentifier {
```

Comment:
Can you copy over the test `Eth1AddressSignerIdentifierTest` as well

Reply:
done